docs/project/find-an-issue.md

```diff
@@ -107,9 +103,9 @@ In this section, you find and claim an open documentation lines issue.
 
     ![Open issues](/project/images/issue_list.png)
 
-3. Look for the <strong class="gh-label beginner">exp/beginner</strong> items on the list.
+3. Look for the <strong class="gh-label beginner">exp/beginner</strong> issues.
```

Comment:
I wonder if this step could be removed completely? Once the exp/beginner label is applied in step 4 the user will only see the relevant issues anyway.  Any thoughts?